src/ui/AddPhotoBlock/AddPhotoBlock.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './AddPhotoBlock.scss';
+
+const AddPhotoBlock = ({ photo, setPhoto, sizeLimit, lagelText }) => {
+  const handlePhoto = (e) => {
+    if (e.target.files[0].size < sizeLimit) {
+      const fileReader = new FileReader();
+      fileReader.onload = () => {
+        setPhoto(fileReader.result);
+      };
+      fileReader.readAsDataURL(e.target.files[0]);
+    } else {
+      setPhoto('');
+    }
+  };
+
+  return (
+    <div className='add-photo-block'>
+      <label className='add-photo-block__label'>{lagelText}</label>
+      <div className='add-photo-block__photo-place'>
+        <label>
+          <input onChange={handlePhoto} type='file' accept='.jpg, .jpeg, .png, .bmp' multiple={false} />
+          <img className={`add-photo-block__photo ${!photo && 'add-photo-block__photo_hidden'}`} src={photo} alt='' />
+        </label>
```

Comment:
```suggestion
    <label className='add-photo-block__label' htmlFor={inputId}>
        {labelText}
      </label>
      <div className='add-photo-block__photo-place'>
        <label>
          <input
            id={inputId} 
        
        
```


для тега label лучше добавить значимый атрибут htmlFor, соответствующий id поля ввода. Это связь гарантирует, что щелчок по метке активирует поле выбора файла.

Reply:
@stillNiceTanya, спасибо, добавил